code/__DEFINES/job.dm

```diff
@@ -30,6 +30,22 @@ var/global/list/job_squad_roles = JOB_SQUAD_ROLES_LIST
 #define JOB_SYNTH_SURVIVOR				"Synth Survivor"
 #define JOB_CO_SURVIVOR				"Commanding Officer Survivor"
 
+#define ANY_SURVIVOR					"Any Survivor"
+#define CIVILIAN_SURVIVOR				"Civilian Survivor"
+#define SECURITY_SURVIVOR				"Security Survivor"
+#define SCIENTIST_SURVIVOR				"Scientist Survivor"
+#define MEDICAL_SURVIVOR				"Medical Survivor"
+#define ENGINEERING_SURVIVOR			"Engineering Survivor"
+#define CORPORATE_SURVIVOR				"Corporate Survivor"
+#define SURVIVOR_VARIANT_LIST			list(ANY_SURVIVOR, CIVILIAN_SURVIVOR, SECURITY_SURVIVOR, SCIENTIST_SURVIVOR, MEDICAL_SURVIVOR, ENGINEERING_SURVIVOR, CORPORATE_SURVIVOR)
+
+#define SPAWN_PRIORITY_VERY_HIGH		1
+#define SPAWN_PRIORITY_HIGH				2
+#define SPAWN_PRIORITY_MEDIUM			3
+#define SPAWN_PRIORITY_LOW				4
+#define SPAWN_PRIORITY_VERY_LOW			5
+#define LOWEST_SPAWN_PRIORITY			5
```

Comment:
```suggestion
#define SPAWN_PRIORITY_VERY_LOW			5
#define LOWEST_SPAWN_PRIORITY			SPAWN_PRIORITY_VERY_LOW
```

Reply:
the defines are not supposed to be the same, while the numbers are the same, these can be changed in the future if someone decides to.